src/odb/include/odb/db.h

```diff
@@ -7583,8 +7583,13 @@ class dbPowerSwitch : public dbObject
   void setOutSupplyPort(const std::string& out_port);
   void addControlPort(const std::string& control_port);
   void addOnState(const std::string& on_state);
+  void addLibCell(const std::string& lib_cell);
+  void addPortMap(const std::string& model_port,
+                  const std::string& switch_port);
```

Comment:
this should take in the dbMTerm for the terminal that maps to a master

Reply:
same as above